src/filters/sort.js

```diff
@@ -0,0 +1,9 @@
+const sortedNotes = (notes, sortBy) => {
+  if (sortBy === "CREATION_TIME_OLDER")
```

Comment:
I think OLD_TO_NEW or NEW_TO_OLD would be more appropriate naming here. Just a suggestion.